### PR TITLE
fixing errors when undifined return and missing element, adding provi…

### PR DESCRIPTION
**Pull request by BBriset**

### CODE REVIEW

**Review by BenZen59 — DISMISSED** [review]

File: src/components/FicheFilm/FicheFilm.css [hunk 1]
```diff
@@ -1,3 +1,4 @@
+
```
Comment (LoicBrassart) [inline src/components/FicheFilm/FicheFilm.css]: Useless empty line

File: src/components/FicheFilm/FicheFilm.jsx [hunk 2]
```diff
@@ -74,42 +97,82 @@ export default function FicheFilm() {
     UsersScorePictures = star1;
   }
 
-  const hour = (runtimes - (runtimes % 60)) / 60;
-  const min = (runtimes / 60 - hour) * 60;
+  const Runtime = () => {
+    if (runtimes > 59) {
+      const hour = (runtimes - (runtimes % 60)) / 60;
+      const min = Math.round((runtimes / 60 - hour) * 60 * 100) / 100;
```
Comment (LoicBrassart) [inline src/components/FicheFilm/FicheFilm.jsx]: `min = runtime % 60`, and I'm almost sure I already checked this code on another PR

**Review by LoicBrassart — DISMISSED** [review]
Minor Evils to fix

**Review by BenZen59 — DISMISSED** [review]

File: src/components/FicheFilm/FicheFilm.css [hunk 3]
```diff
@@ -228,13 +239,25 @@ img {
...
+  }
+
   @media screen and (max-width: 768px) {
     .starScore {
       width: 120px;
       height: 22px;
     }
 
-    .titleVote {
+    .titreVote {
```
Comment (Hacene010) [inline src/components/FicheFilm/FicheFilm.css]: Franglish ???

**Review by Hacene010 — CHANGES_REQUESTED** [review]
Why did you change the title to titre ??

**Review by BenZen59 — DISMISSED** [review]

**Review by Hacene010 — DISMISSED** [review]

**Comment by BBriset** [comment]
merging conflict resolved

**Review by BenZen59 — DISMISSED** [review]

**Review by BenZen59 — APPROVED** [review]

**Review by Hacene010 — APPROVED** [review]

**Pull request merged**